noun/vortex.c

```diff
@@ -102,6 +102,8 @@ u3v_start(u3_noun now)
     printf("arvo: time: %s\n", wen_c);
     free(wen_c);
   }
+
+  u3t_trace_open();
```

Comment:
Does this need to be inside a `u3C.wag_w & u3o_trace` condition?

Reply:
yes lol

Reply:
fixed!